### PR TITLE
guide for starting fresh

### LINKED ISSUE #251: ⭐ [Documentation] - Getting Started article
### Description

It's confusing where to find information about starting to use Actual.

- The Getting Started section is all about installing the software, not using it to manage your finances.
- The parent page of Tour of Actual says "This article covers the first steps you’ll likely want to take when you first open Actual" and has good setup content. However, all the sub-pages are a UI overview which feels unrelated to this parent.
- The Using Actual section is mostly about steady-state and doesn't address setup.

I propose the following changes:
- Rename "Getting Started" to something more specific to installation and software setup.
- Move and expand the content from the Tour of Actual parent page to a new first article under Using Actual titled "Getting Started."

This new page should address things like initially adding accounts and setting up budget categories, with links off to other pages where more detailed topics may come up (such as starting off with pre-existing credit card debt).

### Documentation URL

https://actualbudget.org/docs/tour/

### PR DESCRIPTION
**Pull request by youngcw**
Depends on #278.
Fixes #251

This adds a dedicated guide for those that are starting fresh with Actual and aren't familiar with zero-sum budgeting.  It is a step by step that points to the existing pertinent docs pages with additional direction.

### CODE REVIEW

File: docs/getting-started/starting-clean.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+### Credit Cards
+If you currently are carrying credit card debt in an on budget account you will need to capture that debt in a category.
+A guide on how to handle that can be found [here](../budgeting/credit-cards)
+
+### The Month Ahead Method
+It can be hard to know where to budget your funds when you get paid in the middle of the month, or get multiple paychecks per month.
+One way to handle this is called the "month ahead" method.
+This consists of holding everything you make this month and only budgeting it next month.
+The goal is to not need any of this month's income for this month's bills, but pay all of this month's bills with last month's income.
+Actual makes this easy by allowing you to hold your available funds for the next month by clicking the to budget amount
```
Comment (shall0pass) [inline docs/getting-started/starting-clean.md:86]:
```suggestion
Actual makes this easy by allowing you to hold your available funds for the next month by clicking the **To Budget** amount at the top of the budgeting screen
```

File: docs/getting-started/starting-clean.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+
+## 4. Keep Budgeting
+Now that you have setup your budget you can start importing transactions and tracking your spending and saving.
```
Comment (shall0pass) [inline docs/getting-started/starting-clean.md:90]:
```suggestion
Now that you have set up your budget, you can start importing transactions and tracking your spending and saving.
```

File: docs/getting-started/starting-clean.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+### How to Setup Accounts
+The process of adding an account is detailed [here](../accounts/#adding-a-new-account).
+
+For bank syncing, Actual currently has built in support for [goCardless](../advanced/bank-sync) which works for most EU/UK banks.
+For other bank syncing option see the [community projects page](../community-repos).
+
+## 2. Setting up your Budget Categories
+
+Now that your accounts are setup and your current balance is accurate, you can start adding budget categories.
```
Comment (shall0pass) [inline docs/getting-started/starting-clean.md:31]:
```suggestion
Now that your accounts are set up and your current balance is accurate, you can start adding budget categories.
```

**Review by shall0pass — COMMENTED** [review]
I used https://www.grammarbook.com/blog/spelling/setup-vs-set-up/ as a reference on when to use setup vs set up and grammarcheck.net to help with the punctuation.

File: docs/getting-started/starting-clean.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+---
+For most users it's best to start fresh with a blank file.
+This guide will walk through setting up a budget file fresh without migrating from a previous budget software export.
+
+## 1. Setting up Accounts
+It is recommended to add all accounts you have to Actual including all savings, checking, and investment accounts.
+These accounts should match what shows up in your Bank or Credit Union.
+For example, if you have a savings, checking, and credit card account with Bank of America, add each as a separate account in Actual.
+### On or Off Budget
+On budget accounts are included towards the funds available in your budget, while off budget accounts are only for viewing or tracking and ease of transfers.
```
Comment (MatissJanis) [inline docs/getting-started/starting-clean.md:12]:
```suggestion
On budget accounts are included towards the funds available in your budget, while off budget accounts are only for tracking.
```

File: docs/getting-started/starting-clean.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+Off budget accounts are included in the net worth report.
+It is usually best to err towards putting accounts on budget versus off.
+The most common off budget accounts would be investment type accounts such as a 401(k), IRA, HSA, brokerage account, loans, and asset tracking such as your home and car value.
```
Comment (MatissJanis) [inline docs/getting-started/starting-clean.md:15]:
```suggestion
The most common off budget accounts would be investment type accounts such as a 401(k), IRA, HSA, brokerage account, loans, and asset tracking such as your home equity.
```

does anyone really track their car value in a budgeting app? Seems like a very narrow edge case, so I'd recommend removing it.

File: docs/getting-started/starting-clean.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+Savings accounts can be off budget, but it is recommended to put them on budget.
+That way you can use the budget categories to view where you plan to use those savings funds in the future.
+
+In some situations it may make sense to put a credit card account off budget. This is generally not recommended unless you are not using that card for any spending and are exclusively paying it off.
+
+### How to Setup Accounts
+The process of adding an account is detailed [here](../accounts/#adding-a-new-account).
```
Comment (MatissJanis) [inline docs/getting-started/starting-clean.md:23]:
```suggestion
The process of adding an account is detailed in [the adding a new account page](../accounts/#adding-a-new-account).
```

best to give links meaningful names.

File: docs/getting-started/starting-clean.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+For bank syncing, Actual currently has built in support for [goCardless](../advanced/bank-sync) which works for most EU/UK banks.
```
Comment (MatissJanis) [inline docs/getting-started/starting-clean.md:25]:
```suggestion
For bank syncing, Actual has built-in support for [GoCardless](../advanced/bank-sync) which works for most EU/UK banks.
```

File: docs/getting-started/starting-clean.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+For other bank syncing option see the [community projects page](../community-repos).
+
+## 2. Setting up your Budget Categories
+
+Now that your accounts are set up and your current balance is accurate, you can start adding budget categories.
+As a rule of thumb, make as many categories as you can think of.
```
Comment (MatissJanis) [inline docs/getting-started/starting-clean.md:31]:
```suggestion
```

I don't think this is a good rule of thumb. People should not go crazy and create hundreds of categories. One for each vendor.

File: docs/getting-started/starting-clean.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+If you have an investment account that you contribute to, make a category for that.
+
+You also have the ability to make multiple income categories.
+Maybe you have multiple income streams, make a category for each one.
+You may also want a category for interest and dividends.
+
+### How to Setup Categories
+While making these categories, remember that all money entering or leaving your budget needs a category.
+
+The process for adding and working with categories is found [here](../budgeting/categories/#add-a-category).
```
Comment (MatissJanis) [inline docs/getting-started/starting-clean.md:48]:
```suggestion
The process for adding and working with categories is found in [the category guide](../budgeting/categories/#add-a-category).
```

File: docs/getting-started/starting-clean.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+Those important categories might be rent, food, and utilities.
+Assign what you need for those categories, then repeat the process for the next most important categories until your to budget amount for the month reaches zero.
+When you get a paycheck or other income continue filling in where the funds are needed.
+
+As you go along you will start to get a good view of where your money is going.
+If you don't like how things are going, make changes.
+If you're happy, great!
+Keep budgeting to keep track of your progress on your goals.
+
+A more detailed breakdown of how budgeting in Actual works is provided [here](../budgeting/)
```
Comment (MatissJanis) [inline docs/getting-started/starting-clean.md:74]:
```suggestion
A more detailed breakdown of how budgeting in Actual works is provided in [the budgeting page](../budgeting/)
```

File: docs/getting-started/starting-clean.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+### Credit Cards
+If you currently are carrying credit card debt in an on budget account you will need to capture that debt in a category.
+A guide on how to handle that can be found [here](../budgeting/credit-cards)
```
Comment (MatissJanis) [inline docs/getting-started/starting-clean.md:78]:
```suggestion
A guide on how to handle that can be found in [the credit card guide](../budgeting/credit-cards)
```

**Review by MatissJanis — APPROVED** [review]
Overall LGTM! Just some small nitpicks. Feel free to ignore them.

**Pull request merged**